### PR TITLE
feat(number input): prevent input change on mouse wheel

### PR DESCRIPTION
**Pull request by ddouglasz**
#### Summary

Fixes https://github.com/commercetools/ui-kit/issues/2740

## Description

If a number input is focused and you scroll, the value of the input is changed inadvertently as can be seen below. We want to be able to scroll with the mouse without having it affect the number input, whether it's focused or not.

### CODE REVIEW

**Review by kark — APPROVED** [review]
Thanks Douglas 🙌 Looks good to me!

File: .changeset/calm-moons-run.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@commercetools-uikit/number-input': minor
+---
+
+prevent number input change when scrolling with the mouse.
```
Comment (CarlosCortizasCT) [inline .changeset/calm-moons-run.md:5]:
```suggestion
Preventing the value of the input to be updated when it has the focus and the user scrolls the page content using the mouse wheel.
```

**Review by CarlosCortizasCT — APPROVED** [review]
Thanks 👍

File: .changeset/calm-moons-run.md (hunk 1 above)
Comment (ddouglasz) [reply .changeset/calm-moons-run.md]: updated here: https://github.com/commercetools/ui-kit/pull/2750/commits/5020ad798c4a22d7ab62fb41ec3a68391b43f509`

Thank you.

**Pull request merged**